lib/theme/components/_modals.scss

```diff
@@ -43,6 +61,14 @@ ion-modal .ion-page {
       padding: 0;
     }
 
+    &-header__title-container {
+      @include ms.responsive-breakpoint('sm') {
+        padding: 1.5rem 2rem;
+        margin-bottom: 1rem;
+        border-bottom: 1px solid var(--parsec-color-light-secondary-medium);
```

Comment:
Is this the expected outcome? 
![Capture d’écran 2025-04-30 à 15 28 31](https://github.com/user-attachments/assets/c2270c20-cd46-4c5b-8861-12f6bd496ca1)